src/components/cloud-cover/cloud-cover.twig

```diff
@@ -1,12 +1,26 @@
+{% set _heading_block %}{% block heading %}{% endblock %}{% endset %}
+{% set _content_block %}{% block content %}{% endblock %}{% endset %}
 {% set _extra_block %}{% block extra %}{% endblock %}{% endset %}
 {% set _scene_block %}{% block scene %}{% endblock %}{% endset %}
 
 <div class="c-cloud-cover t-dark{% if class %} {{ class }}{% endif %}">
   <div class="o-container o-container--pad-inline">
     <div class="o-container__content c-cloud-cover__inner">
       <div class="c-cloud-cover__content">
-        {% block content %}
-        {% endblock %}
+        {% embed '@cloudfour/objects/rhythm/rhythm.twig' with {
+          class: 'o-rhythm--condensed',
+          _heading_block: _heading_block,
+          _content_block: _content_block
+        } only %}
+          {% block content %}
+            <div class="c-cloud-cover__heading">
+             {{ _heading_block }}
+            </div>
+            <div class="c-cloud-cover__copy">
+             {{ _content_block }}
+            </div>
```

Comment:
This element should probably be a rhythm object so that multiple paragraphs will still have spacing between them.

(Note that it shouldn't be necessary to set the rhythm to `condensed`, as now the rhythm object will inherit the last rhythm value automatically.)